_posts/2017-11-08-cost-of-drupal-7.md

```diff
@@ -0,0 +1,180 @@
+
+## Our advice
+
+So, we've dumped a lot of information on you at this point, but it may still not be entirely clear what you should do with your outdated Drupal site. Ahead we provide general suggestions as well as what is pertinent for each version separately (Drupal 6 and 7) to site owners.
+
+### To everyone on Drupal <8
+
+We still have <3 for Drupal <8. It got us to where we are. But here's what we think you should consider about where you're going.
+
+1. **Plan 3 years out** if possible. A stitch, in time, saves nine, and every minute of planning saves 10 in execution. It's best to be in the driver's seat on these large investment decisions rather than at the mercy of fleeting security support. An experienced partner will help shepherd this process, but be vigilant regardless. Create a roadmap that accounts for substantial functional changes, a redesign, an integration to another system, or whatever is unique to your organization. This will help you use your resources most efficiently. It's understandably not as compelling to site owners and budget stakeholders to invest just for the sake of upgrading. The security implications will often be undervalued and overlooked. If your organization hasn't historically done this, it doesn't help you much in the present, but that's all the more reason to change your orientation now.
+1. **Be mindful of what you _don't_ need**. We all get excited about the possibilities of new functionality. However, when things we've built have outlived their purpose, let them go. Given the complexity and interdependence of the tools we build, customizations take the form of mounting, insidious and potentially crippling technical debt if left unaddressed. The cost to upgrade the technical debt is likely the major cause for most of those who have not yet upgraded; it is certainly the case for all of our partners who haven't. This debt can be hard to track, and it's not something most agencies proactively share since they have a hand in creating it and maintain job security to maintain it. Ask for answers as to how your partner is managing your technical debt. If you don't get good answers, keep asking. Another subset of this concept is that even if you want to maintain certain functionality, it needn't be done in the with the same modules on the Drupal 8 platform. So don't take a given module's absence in Drupal 8 as a certainty that it cannot be efficiently achieved in Drupal 8. In many cases, it can.
+1. **Training will be required.** If you plan to build with the same team that built your <8 site, and they have not worked on any other Drupal 8 or object-oriented PHP projects, make sure you dedicate time and budget resources for substantial training.
+
+### To those with production Drupal 6 sites.
+
```

Comment:
I don't have a lot of faith in migrations for sites with anything but the most simple configuration in D6, so I don't know if I would make "What has changed is the maturity of the migration, making it easier to migrate from 6->8. This should be where you’re thinking based on all of the above, and the job before going forward should be to well vet all assumptions." the main focus